audio/audiostream.h

```diff
@@ -305,37 +331,45 @@ class SubSeekableAudioStream : public SeekableAudioStream {
 	Timestamp _pos;
 };
 
+/**
+ * A QueuingAudioStream class that allows for queuing audio streams for playback.
+ * @todo A bit more might be needed here
```

Comment:
I think it is fine like this. Although maybe we can highlight its specificity a bit more by adding `multiple`:
`A QueuingAudioStream class that allows for queuing multiple audio streams for playback.`